src/redemptions/redemption.service.ts

```diff
@@ -255,6 +279,7 @@ export class RedemptionService {
         id_details: instanceToPlain(data.idDetails),
         failure_message: data.failureMessage,
         public_address: data.publicAddress,
+        ...(data.age ? { age: data.age } : {}),
```

Comment:
Aren't undefined filtered out by the prisma filter?

```suggestion
        data: data.age
```

Reply:
https://github.com/iron-fish/ironfish-api/pull/1356#discussion_r1136147718